check.go

```diff
@@ -67,13 +76,117 @@ func (c *CheckRunner) Stop() {
 	}
 }
 
+// Update an HTTP check
+func (c *CheckRunner) updateCheckHTTP(latestCheck *api.HealthCheck, checkHash types.CheckID, definition *api.HealthCheckDefinition, updated checkIDSet, added checkIDSet) bool {
```

Comment:
Not going to require this to be fixed, but a 178 character code line is way to long.

Reply:
Thanks for this feedback! Yes, I think we could improve readability here. @findkim mentioned this to me as well suggesting I could clean up duplicate types. I'll do that and add a newline after `checkHash types.CheckID`. I'll do this for `updateCheckTCP` as well and something similar for `NewCheckRunner`